CryptoPrices/Sources/Home/Sources/Home/WalletStatisticSection.swift

```diff
@@ -53,26 +58,26 @@ private extension WalletStatisticSection {
     var todayProfit: some View {
         HStack {
             VStack(alignment: .leading, spacing: 8.0) {
-                Text(Strings.Home.todayProfitTitle)
-                    .foregroundColor(.lightSilver)
-                    .font(.Inter.medium.font(for: .callout))
+                Text(Strings.Home.TodayProfit.title)
+                    .foregroundColor(Assets.Colors.lightSilver.swiftUIColor)
```

Comment:
This is optional 🙌 . I feel it inconvenient to access Assets of color & image types a little bit 😄 . I think we can improve it a bit by:

``` Swift
typealias Colors = Assets.Colors
typealias Images = Assets.Images

=> Colors.lightSilver.swiftUIColor
```

Reply:
the same with .font 😄 below, I think it could be improved. But it's also fine to be kept as the current one. Not big deal :D.

Reply:
one upvote from me to use `typealias` for cleaner code 👍

Reply:
For the `Colors` and `Images` I intended to make them under the namespace `Assets` since they are actually the types of _assets_ (the resources are even put into files with the `xcassets` extension)
But now I see them quite redundant and lengthy too 😄 

Will update the `swiftgent.yml` to generate without the `Assets` namespace (so no need to add `typealias`)

Reply:
For the font could you elaborate more 🙏 

As I see now, syntax like `.font(Fonts.Inter.bold)` is quite not too short but is fine (to me) and descriptive. 
This is because it's normal the app contains more than 1 font family (our simple app may contains `Inter` only)
So I think we better explicitly here instead of removing the font name (to become `.font(Fonts.bold)` or `.font(.bold)`)